src/utils/specs.py

```diff
@@ -52,129 +53,264 @@
+            'name': 'GPU Model'})
+    gpu_arch: str = field(
+        default=None,
+        metadata={
+            'doc': 'The architecture name of the accelerators/GPUs in the system,\n'
+                   'as used by (e.g.,) the AMDGPU backed of LLVM.',
+            'name': 'GPU Arch'})
+    gpu_l1: str = field(default=None, metadata={'doc': 
+        "The size of the vL1D cache (per compute-unit) on the accelerators/GPUs in the system in KiB",
+        'name': 'GPU L1'})
+    gpu_l2: str = field(default=None, metadata={'doc': 
+        "The size of the vL1D cache (per compute-unit) on the accelerators/GPUs in the system in KiB",
+        'name': 'GPU L2'})
+    cu_per_gpu: str = field(default=None, metadata={'doc':
+        "The total number of compute units per accelerator/GPU in the system. On systems with configurable\n"
+        "partitioning, (e.g., MI300) this is the total number of compute units in a partition.",
+        'name': 'CU per GPU'})
+    simd_per_cu: str = field(default=None, metadata={'doc':
+        "The number of SIMD processors in a compute unit for the accelerators/GPUs in the system.",
+        'name': 'SIMD per CU'})
+    se_per_gpu: str = field(default=None, metadata={'doc': 
+        'The number of shader engines on the accelerators/GPUs in the system. On systems with configurable\n'
+        'partitioning, (e.g., MI300) this is the total number of shader engines in a partition.',
+        'name': 'SE per GPU'})
+    wave_size: str = field(default=None, metadata={'doc': 
+        'The number work-items in a wavefront on the accelerators/GPUs in the system.',
+        'name': 'Wave Size'})
+    workgroup_max_size: str = field(default=None, metadata={'doc': 
+        'The maximum number of work-items in a workgroup on the accelerators/GPUs in the system.',
+        'name': 'Workgroup Max Size'})
+    max_waves_per_cu: str = field(default=None, metadata={'doc': 
+        'The maximum number of wavefronts that can be resident on a compute unit on the\n'
+        'accelerators/GPUs in the system',
+        'name': 'Max Waves per CU'})
+    max_sclk: str = field(default=None, metadata={'doc': 
+        'The maximum engine (compute-unit) clock rate of the accelerators/GPUs in the system.',
+        'name': 'Max SCLK',
+        'unit': 'MHz'})
+    max_mclk: str = field(default=None, metadata={'doc': 
+        'The maximum memory clock rate of the accelerators/GPUs in the system.',
+        'name': 'Max MCLK',
+        'unit': 'MHz'})
+    cur_sclk: str = field(default=None, metadata={'doc': 
+        '[RESERVED] The current engine (compute unit) clock rate of the accelerators/GPUs in the system. Unused.',
+        'name': 'Max SCLK',
+        'unit': 'MHz'})
+    cur_mclk: str = field(default=None, metadata={'doc': 
+        '[RESERVED] The current memory clock rate of the accelerators/GPUs in the system. Unused.',
+        'name': 'Max MCLK',
+        'unit': 'MHz'})
```

Comment:
```suggestion
    max_sclk: str = field(default=None, metadata={'doc': 
        'The maximum engine (compute-unit) clock rate of the accelerators/GPUs in the system.',
        'name': 'Max SCLK',
        'unit': 'MHz'})
    max_mclk: str = field(default=None, metadata={'doc': 
        'The maximum memory clock rate of the accelerators/GPUs in the system.',
        'name': 'Max MCLK',
        'unit': 'MHz'})
    cur_sclk: str = field(default=None, metadata={'doc': 
        '[RESERVED] The current engine (compute unit) clock rate of the accelerators/GPUs in the system. Unused.',
        'name': 'Cur SCLK',
        'unit': 'MHz'})
    cur_mclk: str = field(default=None, metadata={'doc': 
        '[RESERVED] The current memory clock rate of the accelerators/GPUs in the system. Unused.',
        'name': 'Cur MCLK',
        'unit': 'MHz'})
```
I think this is what you meant to say?

Reply:
Ah, yes -- thanks